core/src/components/divider/test/basic/divider.e2e.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ config, screenshot, title }) => {
+  test.describe(title('divider: rendering'), () => {
+    test('should not have visual regressions', async ({ page }) => {
+      await page.goto('/src/components/divider/test/basic', config);
```

Comment:
Instead of taking a screenshot of the entire page, we should use `setContent` to take screenshots of just the divider in different scenarios. You can find an example here: https://github.com/ionic-team/ionic-framework/blob/40932d7b2d9f6650ed621e0183b9c70b74debff8/core/src/components/avatar/test/shape/avatar.e2e.ts#L11-L16